importer/generate.js

```diff
@@ -78,7 +82,7 @@ function processFolder(srcPath, destPath) {
 
         var destFile = path.join(destPath, file);
         fs.copyFileSync(srcFile, destFile);
-
+        var iconContent = fs.readFileSync(srcFile, { encoding: "utf8"})
```

Comment:
Since this is only read in one branch, I'd move this down to the `generateReact` method.